pkg/cmd/uninstall/uninstall.go

```diff
@@ -236,6 +236,13 @@ func (o *UninstallOptions) deleteNamespace(namespace string) error {
 // of an unrelated error
 func (o *UninstallOptions) DeleteReleaseIfPresent(ns string, releaseName string, errors []error, force bool) []error {
 	if err := o.Helm().StatusRelease(ns, releaseName); err == nil || force {
+
+		err = o.setPVCFinalizerToNull(ns, releaseName)
+
+		if err != nil {
+			errors = append(errors, fmt.Errorf("failed to uninstall the %s helm chart in namespace %s: %s", releaseName, ns, err))
```

Comment:
This error actually comes from the `setPVCFinalizerToNull`, the error message should be related.